js/core/src/registry.ts

```diff
@@ -115,21 +115,30 @@ export class Registry {
   private allPluginsInitialized = false;
   public apiStability: 'stable' | 'beta' = 'stable';
 
-  readonly asyncStore = new AsyncStore();
-  readonly dotprompt = new Dotprompt({
-    schemaResolver: async (name) => {
-      const resolvedSchema = await this.lookupSchema(name);
-      if (!resolvedSchema) {
-        throw new GenkitError({
-          message: `Schema '${name}' not found`,
-          status: 'NOT_FOUND',
-        });
-      }
-      return toJsonSchema(resolvedSchema);
-    },
-  });
-
-  constructor(public parent?: Registry) {}
+  readonly asyncStore: AsyncStore;
+  readonly dotprompt: Dotprompt;
+
+  constructor(public parent?: Registry) {
```

Comment:
Please declare variable separately (I want to turn on "erasable syntax only" sometime)

Reply:
Done.